### PR TITLE
Prep 5.1.0 release

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Prep 5.1.0 release

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

several

### CODE REVIEW

**Review by tremble — APPROVED** [review]

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -1406,3 +1406,111 @@ releases:
...
+    - 1172-credentials_parameters.yml
+    - 20221010-test-times.yml
+    - 20221021-ec2_instance-tower_callback.yml
+    - 20221026-pytest-forked.yml
+    - 20221027-ec2_security_group-nested.yml
+    - DEPRECATE-module_utils-urls.yml
+    - summary.yml
+    - version_added-5.yml
+    - workflows-add-intersphinx.yml
+    release_date: '2022-10-27'
```
Comment (tremble) [inline changelogs/changelog.yaml:1516]:
```suggestion
    release_date: '2022-10-28'
```

**Comment by tremble** [comment]
#1219 is ready to merge (will need to remove the sanity ignores files) once we've finished the 5.1.0 release

**Comment by alinabuzachis** [comment]
@tremble these ones seems unnecessary. 

```
ERROR: Found 2 pep8 issue(s) which need to be resolved:
--
1027 | 2022-10-28 09:34:54.750472 \| controller \| ERROR: tests/sanity/ignore-2.14.txt:1:1: A100: Ignoring 'E275' on 'plugins/inventory/aws_ec2.py' is unnecessary
1028 | 2022-10-28 09:34:54.750477 \| controller \| ERROR: tests/sanity/ignore-2.14.txt:3:1: A100: Ignoring 'E275' on 'plugins/modules/s3_object_info.py' is unnecessary

```

**Comment by alinabuzachis** [comment]
@tremble can we merge it now?

**Comment by tremble** [comment]
Yup, I was leaving it to you.

**Pull request merged**